Woof/Woof/View/DetailedPetSitterView/ContactDelailsSectionView.swift

```diff
@@ -0,0 +1,28 @@
+import SwiftUI
+
+/// A view displaying a section for the detailed sitter's information.
+struct ContactDelailsSectionView: View {
```

Comment:
Would we use this only to display info for sitter?
It looks like a reusable element, where we can provide imageName for an icon and specify the text to show.

Reply:
Yes, agree. Thanks, fixed.